starter/script.js

```diff
@@ -1 +1,140 @@
+    if (cap === 'yes') {
+      alert("Phew, your options are wide open. You could get a twist-top but what fun is that? Be a civilized person and get a corked bottle.");
+
+      // fifth prompt:
+      var identification = prompt("The manager just stopped you to ask for some ID. Are you going to show your 'license' or create a 'diversion'?");
+      if (identification === 'license') {
+        alert("Nothing to see here, just an over-21-adult doing adult wine things. You're good to go!");
+
+        // sixth prompt:
+        var pick = prompt("Hey, there's a display over here with the employee pick of the month, and it sounds perfect. Will you make a safe choice and get the 'display' wine, choose that 'flashy' label with all the French writing, or just get your 'usual'?");
+        // ENDPATH5
+        if (pick === 'display') {
+          alert("Crowd favorite! But wait.. the employees love this wine so much that they all abandon their registers and throw down with you right there in the store! One thing leads to another, and it looks like you're sleeping in your car tonight.");
+        }
+        // ENDPATH6
+        else if (pick === 'usual') {
+          alert("Uh oh, your friends heard what you picked and are not excited to drink your usual for the millionth time. You'll enjoy your evening, but it looks like it'll be alone this time.");
+        }
+        else {
+          alert("Way to be brave. Time to go home and try it out");
+
+          // seventh prompt, loop:
+          var numGlasses = parseInt(prompt("You definitely made the right choice, this wine is so delicious that you could probably finish a whole bottle by yourself. How many glasses have you had, anyway?"));
+          while(numGlasses < 6) {
+            if (numGlasses < 2) {
+              alert("Party's just getting started, refill that cup!");
+            }
+            else if (numGlasses < 4) {
+              alert("Ok you might be responsible for that spill on the carpet but go ahead, that bottle's not empty.");
+            }
+            else {
+              alert("You could get one last pour out of this bottle, go for it!");
+            }
+            numGlasses = parseInt(prompt("Wine night has been winding down, might want to check the supply levels. How many drinks have you had now?"));
+          }
+
+          //ENDPATH7
+          alert("Good news is you found a new favorite. Bad news is you finished a whole bottle PLUS some, and your friends cut you off. Until next time!");
+        }
+
+      }
+      // >ENDPATH4
+      else {
+        alert("DANGIT! You've been caught.. ditch the basket and shout- THIS ISN'T TACO BELL!! WHERE AM I?!? -and make a run for the door. Better luck next year, kid.");
+      }
+
+    }
+    //ENDPATH3
+    else {
+      alert("Welp. I don't see any twist-tops here. Looks like your night just ended.");
+    }
+  }
+}
```

Comment:
This was super funny, especially the charity gala joke and the taco bell joke; those were my top faves and I really enjoyed playing this! Your code is well organized and commented helpfully, and cross references your pseudo. That's awesome. Being well organized makes it easier to get more done when you code.

As such I don't have any suggestions to improve your code in its current form. However, a way to rewrite it might involve storing all the strings that are used for alerts and prompts in an object full of really long strings, and then accessing properties of that object depending on the path taken by the user. I can show you what that might look like, at your request. Great work!